Tasks/Npm/npmtoolrunner.ts

```diff
@@ -162,4 +163,21 @@ export class NpmToolRunner extends tr.ToolRunner {
             util.restoreFile(this.projectNpmrc());
         }
     }
+
+    private _logExecResults(exitCode: number, stderr: string){
+        try{
+            console.log("##vso[telemetry.publish area=Packaging;feature=Npm]%s",
+            JSON.stringify({
+                'SYSTEM_JOBID': tl.getVariable('SYSTEM_JOBID'),
+                'SYSTEM_PLANID': tl.getVariable('SYSTEM_PLANID'),
+                'SYSTEM_COLLECTIONID': tl.getVariable('SYSTEM_COLLECTIONID'),
+                'command': tl.getVariable(NpmTaskInput.Command),
+                'arguments': tl.getVariable(NpmTaskInput.CustomCommand),
```

Comment:
I don't see the arguments input in the npm task. Should this be removed?

Reply:
tl.getInput

Reply:
Fixed